WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenuViewModel.swift

```diff
@@ -232,41 +218,6 @@ final class InPersonPaymentsMenuViewModel: ObservableObject {
     }()
 }
 
-// MARK: - Collect payment
-
-private extension InPersonPaymentsMenuViewModel {
-    func collectPayment() {
-        let orderViewModel = EditableOrderViewModel(siteID: siteID)
-        self.orderViewModel = orderViewModel
-        orderViewModel.onFinished = { [weak self] _ in
-            self?.dismissPaymentCollection()
```

Comment:
nit: `dismissPaymentCollection` and `navigationPath`/`navigationPathBeforePaymentCollection` properties could be deleted.

Reply:
True, deleted in 0bc3bf9a78d4ea7f442e2caddd248b8f3d1b2543